webrtc.html

```diff
@@ -9566,7 +9578,39 @@ <h3>RTCPeerConnection Interface Extensions</h3>
       <div>
         <pre class="idl">partial interface RTCPeerConnection {
     Promise&lt;RTCStatsReport&gt; getStats (optional MediaStreamTrack? selector = null);
-};</pre>
+    attribute EventHandler onstatsended;
+          };</pre>
+        <section>
+          <h2>Attributes</h2>
+          <dl data-link-for="RTCPeerConnection"
+              data-dfn-for="RTCPeerConnection" class="attributes">
+            <dt><dfn><code>onstatsended</code></dfn> of type
+              <span class="idlAttrType"><a>EventHandler</a></span></dt>
+            <dd>
+              <p>The event type of this event handler
+                is <code><a>statsended</a></code>.
+              </p>
+              <p>When the lifetime of one or more <a>monitored object</a>s ends, the UA MUST
+                queue a task to run the following steps:
+                <ul>
+                  <li>Create a new RTCStatsReport object called <var>r</var></li>
```

Comment:
nit: Usually in the spec variable names are full words, with the exception of "e" for "event". Prefer "report" to "r".

Reply:
Done.